betree/pmdk/src/lib.rs

```diff
@@ -0,0 +1,192 @@
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
+
+include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+
+use std::sync::{Arc, Mutex};
+use std::os::raw::c_void;
+use std::ptr::NonNull;
+
+#[derive(Debug)]
+//pub struct ptr_to_pmem(Arc<Mutex<*mut c_void>>);
+pub struct ptr_to_pmem(Arc<*mut c_void>);
```

Comment:
Naming should follow rust convention, so CamelCase for structs.

If you haven't used it yet `cargo clippy` is a tool which can show you all these hints.